site/en/blog/related-website-sets/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Related Website Sets - the new name for First-Party Sets in Chrome 117
+description: >
+  Related Website Sets (RWS) is the new name for First-Party Sets and also brings increased flexibility in defining sets.
```

Comment:
It might be worth splitting this to two sentences, because it reads a bit odd now.